presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -40,6 +40,10 @@ Array Functions
     Returns the average of all non-null elements of the ``array``. If there is no non-null elements, returns
     ``null``.
 
+.. function:: array_contains_all(x, y) -> boolean
+
+    Returns true if the array ``x`` contains all the elements of array ``y``.
```

Comment:
Mention about null behavior and also make sure it's consistent with our recent changes to other udfs

Reply:
Is it a better version of:

cardinality(array_instrsect(x, y)) > 0

? if so that'll be the best way to do express it. Since you are using a set, maybe use something to indicate it's a set containment semantics by naming it appropriately - like: array_superset(x, y) where x is a superset of y might be best

Reply:
I think this is equivalent to`cardinality(array_intersect(x,y)) = cardinality(y)`.  `cardinality(array_instersect(x, y)) > 0` would be arrays_overlap

Reply:
ARRAY_INTERSET just returns the distinct element. ARRAY_CONTAINS_ALL will check if array X has all the elements of ARRAY including duplicates.